### PR TITLE
Implement new FindCommand

### LINKED ISSUE #44: Implement Find command

### PR DESCRIPTION
**Pull request by psh12320**
Fixes #44

### CODE REVIEW

**Review by ulricolo7 — COMMENTED** [review]
Could not find anything wrong so LGTM!

File: src/test/java/seedu/address/logic/commands/FindCommandTest.java [hunk 1]
```diff
@@ -5,9 +5,8 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static seedu.address.logic.Messages.MESSAGE_PERSONS_LISTED_OVERVIEW;
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
-import static seedu.address.testutil.TypicalPersons.CARL;
+import static seedu.address.testutil.TypicalPersons.BENSON;
```
Comment (XinghuaJulia) [inline src/test/java/seedu/address/logic/commands/FindCommandTest.java:8]: I am not sure if it is recommended to remove test stubs as less stubs may lead to edge cases not considered. KIV perhaps add more stubs in the future.

File: src/test/java/seedu/address/logic/commands/FindCommandTest.java [hunk 2]
```diff
@@ -29,10 +29,15 @@ public class FindCommandTest {
 
     @Test
     public void equals() {
-        NameContainsKeywordsPredicate firstPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("first"));
-        NameContainsKeywordsPredicate secondPredicate =
-                new NameContainsKeywordsPredicate(Collections.singletonList("second"));
+        //        NameContainsKeywordsPredicate firstPredicate =
```
Comment (XinghuaJulia) [inline src/test/java/seedu/address/logic/commands/FindCommandTest.java]: If not used, kindly remove the commented code to adhere to better code quality.

File: src/test/java/seedu/address/logic/commands/FindCommandTest.java [hunk 3]
```diff
@@ -57,33 +62,49 @@ public void equals() {
     @Test
     public void execute_zeroKeywords_noPersonFound() {
         String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
-        NameContainsKeywordsPredicate predicate = preparePredicate(" ");
+        PersonContainsKeywordsPredicate predicate =
+                new PersonContainsKeywordsPredicate("someone", null, null, null, null, null, null);
+        // NameContainsKeywordsPredicate predicate = preparePredicate(" ");
```
Comment (XinghuaJulia) [inline src/test/java/seedu/address/logic/commands/FindCommandTest.java]: Not used commented code can also be removed.

**Review by XinghuaJulia — COMMENTED** [review]
LGTM, minor nits on unused codes that are commented out. Bugs were fixed too, good job :DDDDDD
